7/community/Dockerfile

```diff
@@ -1,20 +1,11 @@
-FROM openjdk:11
-
-ENV SONAR_VERSION=7.9.1 \
-    SONARQUBE_HOME=/opt/sonarqube \
-    # Database configuration
-    # Defaults to using H2
-    # DEPRECATED. Use -v sonar.jdbc.username=... instead
-    # Drop these in the next release, also in the run script
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=""
-
-# Http port
-EXPOSE 9000
-
-RUN groupadd -r sonarqube && useradd -r -g sonarqube sonarqube
+ARG SQ_VERSION=7.9.1
 
+FROM openjdk:11-slim AS build
+ARG SQ_VERSION
+ENV SONAR_VERSION=${SQ_VERSION} \
```

Comment:
do we really need the SQ_VERSION in the build step ? if we can hardcode a folder, it is simpler

Reply:
In my opinion having a version defined in only one place in `Dockerfile` is the best way to avoid typos or version mismatches in multi-stage build. I would be for hardcoding if version was only used in one place.